components/post_view/post_image/index.ts

```diff
@@ -0,0 +1,6 @@
+ // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PostImage from './post_image.js';
```

Comment:
Your `.js` extension here is tripping up the build. You should be able to leave the extension blank.
```suggestion
import PostImage from './post_image';
```